src/idpf.rs

```diff
@@ -80,6 +80,11 @@ impl IdpfInput {
         }
     }
 
+    /// Create a new empty IDPF input.
+    pub fn empty_input() -> IdpfInput {
```

Comment:
More rusty. However since the implementation is a one liner and is pretty readable on its own (`IdfpInput::from_bools(&[])`) I probably would just inline it.
```suggestion
    pub fn new_empty() -> IdpfInput {
```